packages/styled-system/tests/css.test.ts

```diff
@@ -513,15 +513,14 @@ test("returns correct media query 2nd order", () => {
   expect(keys).toMatchInlineSnapshot(`
     Array [
       "flexDirection",
-      "justifyContent",
-      "@media screen and (min-width: 40em)",
-      "@media screen and (min-width: 52em)",
       "color",
       "height",
       "paddingInlineStart",
       "paddingInlineEnd",
       "paddingTop",
       "paddingBottom",
+      "@media screen and (min-width: 40em)",
+      "@media screen and (min-width: 52em)",
```

Comment:
The results of `css()` are as follows.
There are some differences, but I think the behavior will not change.

### Before

```ts
{
  flexDirection: 'column',
  justifyContent: null,
  '@media screen and (min-width: 40em)': {
    justifyContent: 'flex-start',
    paddingInlineStart: 'var(--space-3)',
    paddingInlineEnd: 'var(--space-3)'
  },
  '@media screen and (min-width: 52em)': {
    justifyContent: 'flex-end',
    paddingInlineStart: 'var(--space-4)',
    paddingInlineEnd: 'var(--space-4)'
  },
  color: 'background',
  height: '100%',
  paddingInlineStart: 'var(--space-2)',
  paddingInlineEnd: 'var(--space-2)',
  paddingTop: 'var(--space-4)',
  paddingBottom: 'var(--space-4)'
}
```

### After

```ts
{
  flexDirection: 'column',
  color: 'background',
  height: '100%',
  paddingInlineStart: 'var(--space-2)',
  paddingInlineEnd: 'var(--space-2)',
  paddingTop: 'var(--space-4)',
  paddingBottom: 'var(--space-4)',
  '@media screen and (min-width: 40em)': {
    justifyContent: 'flex-start',
    paddingInlineStart: 'var(--space-3)',
    paddingInlineEnd: 'var(--space-3)'
  },
  '@media screen and (min-width: 52em)': {
    justifyContent: 'flex-end',
    paddingInlineStart: 'var(--space-4)',
    paddingInlineEnd: 'var(--space-4)'
  }
}
```